### DISCUSSION TITLE
Midi out still in development?

### DISCUSSION THREAD

**Discussion by avidteacher** [cpyarger/obs-midi#129] [Q&A] [answered]
Hello,

This is a great plugin for controlling OBS with midi.  Now I want OBS to send Midi to other programs/devices to control them with OBS.   Is this something that is still being developed?

Thanks!
el

**Comment by cpyarger** [comment, marked as answer]
I currently do not have any plans to have the plugin acting like a virtual midi device. Though something may be possible with the upcoming feedback system and something like loopmidi
